ctapipe/io/simteleventsource.py

```diff
@@ -224,7 +224,7 @@ def is_simulation(self):
 
     @property
     def datalevels(self):
-        return (DataLevel.R0, DataLevel.R1, DataLevel.DL0)
+        return (DataLevel.R0, DataLevel.R1)
```

Comment:
does this mean `SimTelEventSource` is not supposed to provide  DL0 ever?

Reply:
I am a bit lost with this line and the title of the PR

Reply:
DL0 was included here by mistake, the `SimTelEventSource` does not currently provide it.

I don't know what it is supposed to provide, and this PR is not about that.

Concerning the title: Currently the DL1 tool needs either DL0 or R1 to be present, so I added a check for this.